WordPressEditor/WordPressEditor/Classes/Plugins/WordPressPlugin/Gutenberg/GutenbergInputHTMLTreeProcessor.swift

```diff
@@ -103,8 +107,12 @@ private extension GutenbergInputHTMLTreeProcessor {
     func openerAttributes(for commentNode: CommentNode) -> [Attribute] {
         let attributeName = GutenbergAttributeNames.blockOpener
         let openerBase64String = encode(commentNode)
-        
-        return [Attribute(name: attributeName, value: .string(openerBase64String))]
+        // TODO: Parse and create an extra attribute to get the block name.        
```

Comment:
I think we should roll back these changes, unless there's a reason to do the extra name processing.